CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # bedrock-oauth2-client ChangeLog
 
+## 2.1.0 - 2021-xx-xx
+
+### Added
+- Added `onInit` parameter to `getAccessToken`. When true, it will continually
```

Comment:
Let's rename this -- it should be named after what it does vs. when or where it's called from. Can we make it just allow us to set the number of retries -- and then set it to `Infinity` when we don't want it to ever stop?